pkgs/by-name/fl/fluxcd-operator/package.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildGo124Module,
+  fetchFromGitHub,
+  installShellFiles,
+  versionCheckHook,
+  nix-update-script,
+  stdenv,
+}:
+buildGo124Module (finalAttrs: {
+  pname = "fluxcd-operator";
+  version = "0.19.0";
+
+  src = fetchFromGitHub {
+    owner = "controlplaneio-fluxcd";
+    repo = "fluxcd-operator";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```

This way we don't accidentally fetch a branch with the same name if one exists

---

Ah thank you, very good point. Updated 👍🏻